installer/tue-install-impl.bash

```diff
@@ -816,14 +810,15 @@ if [ -n "$TUE_INSTALL_SYSTEMS" ]; then
 fi
```

Comment:
I guess it will be a good idea to add tue-install-error pipe to all the functions that are executed at the end like tue-install-system-now if we are going to break out of the installation

Reply:
True.

Reply:
So are you going to do that in this PR?